graphql-ballerina/build.gradle

```diff
@@ -50,59 +59,15 @@ def stripBallerinaExtensionVersion(String extVersion) {
     }
 }
 
-configurations {
-    jbalTools
-}
-
-dependencies {
-    implementation project(":${packageName}-native")
-}
-
-clean {
-    delete "$project.projectDir/target"
-}
-
-jar {
-    manifest {
-        attributes('Implementation-Title': project.name, 'Implementation-Version': project.version)
-    }
-}
-
-task unpackJballerinaTools(type: Copy) {
-    configurations.jbalTools.resolvedConfiguration.resolvedArtifacts.each { artifact ->
-        from zipTree(artifact.getFile())
-        into new File("${buildDir}/target/extracted-distributions", "jballerina-tools-zip")
-    }
-}
-
-task unpackStdLibs() {
-    doLast {
-        configurations.ballerinaStdLibs.resolvedConfiguration.resolvedArtifacts.each { artifact ->
-            copy {
-                from project.zipTree(artifact.getFile())
-                into new File("${buildDir}/target/extracted-distributions", artifact.name + "-zip")
-            }
-        }
-    }
-}
-
-task copyStdlibs(type: Copy) {
-    def ballerinaDist = "build/target/extracted-distributions/jballerina-tools-zip/jballerina-tools-${ballerinaLangVersion}"
-    into ballerinaDist
+apply plugin: 'io.ballerina.plugin'
 
-    /* Standard Libraries */
-    configurations.ballerinaStdLibs.resolvedConfiguration.resolvedArtifacts.each { artifact ->
-        def artifactExtractedPath = "${buildDir}/target/extracted-distributions/" + artifact.name + "-zip"
-        into("repo/bala") {
-            from "${artifactExtractedPath}/bala"
-        }
-        into("repo/cache") {
-            from "${artifactExtractedPath}/cache"
-        }
-    }
+ballerina{
+    module = packageName
+    langVersion = ballerinaLangVersion
+    testCoverageParam = "--code-coverage --jacoco-xml --includes=io.ballerina.stdlib.graphql.*:ballerina.graphql*"
 }
 
-task updateTomlFiles {
+task updateTomlVersions {
```

Comment:
These names are not consistent. Shall we make them consistent? 

Eg.:
- `updateTomlFiles` and `revertTomlFiles` (I prefer this, since it correctly explains what this function actually does) or
- `updateTomlVersions` and `revertTomlVersions` 

Also `revertTomlFile` is misleading, we are updating, reverting multiple toml files here. This might seems like an unnecessary thing, but if we are to make this plugin public, these things matter.

Reply:
Now both tasks are consistent across all the standard libraries, But now it is updateTomlVersions and revertTomlFiles

Reply:
+1 to change these. @ThisaruGuruge shall we open an issue in the plugin-gradle repo? We can add it in the next iteration

Reply:
> Now both tasks are consistent across all the standard libraries, But now it is updateTomlVersions and revertTomlFile

What I meant was the two names of the functions are not consistent with each other. 

> +1 to change these. @ThisaruGuruge shall we open an issue in the plugin-gradle repo? We can add it in the next iteration

Sure, will do.